core/src/main/resources/error/error-classes.json

```diff
@@ -233,6 +243,11 @@
           "Cannot use an UnspecifiedFrame. This should have been converted during analysis."
         ]
       },
+      "WRONG_NUM_ENDPOINTS" : {
+        "message" : [
+          "The number of endpoints must be >= 2 to construct intervals."
```

Comment:
Nit: maybe also include what is current num but this is minor.

Reply:
done